ecommerce/models.py

```diff
@@ -814,3 +826,24 @@ class Meta:
                 condition=models.Q(redeemed=False),
             ),
         )
+
+
+class TaxRate(TimestampedModel):
+    """
+    Stores tax rates for countries. Generally, this should be a VAT rate, but
+    a text field is supplied to store the name of the tax assessed
+    """
+
+    country_code = models.CharField(max_length=2)
+    tax_rate = models.DecimalField(max_digits=6, decimal_places=4)
+    tax_rate_name = models.CharField(max_length=100, null=True, default="VAT")
+    active = models.BooleanField(default=True)
```

Comment:
Thought about this a bit and added a unique on `country_code` - otherwise we'd potentially end up with multiple rates for a country, which wouldn't be useful for us.